gravitee-apim-console-webui/src/management/api/creation-v4/api-creation-v4-spec-stepper-helper.ts

```diff
@@ -168,7 +168,7 @@ export class ApiCreationV4SpecStepperHelper {
     await endpointsConfig.clickValidate();
   }
 
-  async fillAndValidateStep4_1_SecurityPlansList() {
+  async editAndValidateStep4_1_SecurityPlansList() {
```

Comment:
why edit and not keep fillAndValidate like others ?

Reply:
Because nothing was being filled. The others allow a parameter to be passed in the function to "fill" it. 🤷‍♀️ But that can be reverted if it makes more sense as "fill" rather than "edit"